src/wikitextprocessor/parser.py

```diff
@@ -1812,7 +1812,7 @@ def tag_fn(ctx: "Wtp", token: str) -> None:
     # Try to parse it as a start tag
     m = re.match(
         r"""<([-a-zA-Z0-9]+)\s*((\b[-a-zA-Z0-9:]+(\s*=\s*("[^"]*"|"""
-        r"""'[^']*'|[^ \t\n"'`=<>/]*))?\s*)*)(/?)\s*>""",
+        r"""'[^']*'|[^ \t\n"'`=<>]*))?\s*)*)(/?)\s*>""",
```

Comment:
Yeah, this is correct (we might be able to remove other symbols too from the prohibited list, but one step at a time). Tatu is a bit overzealous with being careful, and probably didn't look at the HTML syntax specs. Then again, what is and isn't allowed in HTML has changed over the years so much and been so lax that I don't blame him.

Because "/>" is a token that can't have whitespace inside of it, we can change `(/?)\s*` to `(/?)`. I've been trying to spot these when they come along, and I've removed some of them, but I bet there's still more. Basically, it seems that the HTML tag syntax is that each of `<tagname`, `</endtagname` and `/>` is one token without whitespace inside of it, so `<tagname >`, `</endtagname   >`, `<tagname   />` is allowed, but not `<   tagname>`, `</  endtagname>` or `<tagname /   >`.